api/services/EmailTemplates/forgottenUsername.ejs

```diff
@@ -0,0 +1,3 @@
+<p>
+  Your username for your account at <a href="http://<%= siteUrl %>"><%= siteUrl %></a> is <%= recipientName %>.
```

Comment:
What happens if there are multiple accounts registered for a given email address?